docs/reference/alpha-features.md

```diff
@@ -23,25 +27,23 @@ Characteristics of alpha features and components include:
 - (SaaS) No availability targets.
 - Released outside the standard [release policy](/reference/release-policy.md).
 
-:::note
-
-While there is no maintenance service, customers can still provide feedback through designated support channels, depending on their SLAs. These channels include filing issues in the respective [GitHub repositories](https://github.com/camunda) and by submitting questions and suggestions by [contacting us](/contact).
+To learn more about using alpha features, see [enabling alpha features](/components/console/manage-organization/enable-alpha-features.md).
 
+:::note
+Although there is no maintenance service, customers can still provide feedback through designated support channels, depending on their SLAs. These channels include filing issues in the respective [GitHub repositories](https://github.com/camunda) and by submitting questions and suggestions by [contacting us](/contact).
 :::
 
-Visit our documentation on [enabling alpha features](/components/console/manage-organization/enable-alpha-features.md) to learn more.
-
 ### General availability (GA)
 
 Once features and components are released and considered stable, they become generally available.
 
-Characteristics include:
+Stable features and components are:
 
 - Allowed for production use.
```

Comment:
Given the PR touches this file: @mesellings what is your advice on the word "allowed"?

Content-wise: if a customer uses an alpha for a production use-case, they can do that. They have to accept that all limitations applied - specifically: bugs tickets are considered L3, APIs may break, .... 
Having this in mind, I feel the word "allowed" is a bit strange (but I am not a native speaker). Maybe we should rather say 
`Qualify for L1 Priority-level support (https://docs.camunda.org/enterprise/support/#priority-level) for production use-cases`

Reply:
Changed these bullet points to cover this comment and TW review comments, now reads:

Stable features and components are:

* Ready for production use for most users with minimal risk.
* Supported by [L1 Priority-level support](https://docs.camunda.org/enterprise/support/#priority-level) for production use.
* Fully documented.

A release or component is considered stable if it has passed all verification and test stages and can be released to production.